subprojects/docs/src/docs/userguide/more_about_tasks.adoc

```diff
@@ -481,169 +582,142 @@ For this example, we’re using a custom class to represent the model data. Howe
 
-| `@api:org.gradle.api.tasks.Internal[]`
-| Any type
-| Indicates that the property is used internally but is neither an input nor an output.
-|===
+`link:{javadocPath}/org/gradle/api/tasks/Internal.html[@Internal]` — Expected property type: Any type::
+Indicates that the property is used internally but is neither an input nor an output.
 
-*::
-In fact, `File` can be any type accepted by api:org.gradle.api.Project#file(java.lang.Object)[] and `Iterable&lt;File&gt;` can be any type accepted by api:org.gradle.api.Project#files(java.lang.Object...)[], api:org.gradle.api.file.ProjectLayout#files(java.lang.Object...)[], or api:org.gradle.api.file.ProjectLayout#configurableFiles(java.lang.Object...)[].
+[NOTE]
+====
+[horizontal]
+†::
+In fact, `File` can be any type accepted by link:{groovyDslPath}/org.gradle.api.Project.html#org.gradle.api.Project:file(java.lang.Object)[Project.file(java.lang.Object)] and `Iterable&lt;File&gt;` can be any type accepted by link:{groovyDslPath}/org.gradle.api.Project.html#org.gradle.api.Project:files(java.lang.Object++[]++)[Project.files(java.lang.Object...)], link:{javadocPath}/org/gradle/api/file/ProjectLayout.html#files-java.lang.Object++...++-[ProjectLayout.files(java.lang.Object++...++)], or link:{javadocPath}/org/gradle/api/file/ProjectLayout.html#configurableFiles-java.lang.Object++...++-[ProjectLayout.configurableFiles(java.lang.Object++...++)].
 This includes instances of `Callable`, such as closures, allowing for lazy evaluation of the property values.
 Be aware that the types `FileCollection` and `FileTree` are ``Iterable&lt;File&gt;``s.
+‡::
+Similar to the above, `File` can be any type accepted by link:{groovyDslPath}/org.gradle.api.Project.html#org.gradle.api.Project:file(java.lang.Object)[Project.file(java.lang.Object)]. The `Map` itself can be wrapped in ``Callable``s, such as closures.
+====
 
-**::
-Similar to the above, `File` can be any type accepted by api:org.gradle.api.Project#file(java.lang.Object)[]. The `Map` itself can be wrapped in ``Callable``s, such as closures.
-
-.Additional annotations used to further qualifying property type annotations
-[cols="a,a", options="header"]
-|===
-| Annotation
-| Description
+===== Additional annotations used to further qualifying property type annotations
 
-| anchor:skip-when-empty[]`@api:org.gradle.api.tasks.SkipWhenEmpty[]`
-| Used with `@InputFiles` or `@InputDirectory` to tell Gradle to skip the task if the corresponding files or directory are empty, along with all other input files declared with this annotation. Tasks that have been skipped due to all of their input files that were declared with this annotation being empty will result in a distinct “no source” outcome. For example, `NO-SOURCE` will be emitted in the console output.
+[#skip-when-empty]
+`link:{javadocPath}/org/gradle/api/tasks/SkipWhenEmpty.html[@SkipWhenEmpty]`::
```

Comment:
These should probably be formatted the same way as above (or even incorporated into that table)

Reply:
I've incorporated into the the above table. Please let me know what you think.